lighthouse-core/gather/gatherers/anchor-elements.js

```diff
@@ -37,13 +37,23 @@ function collectAnchorElements() {
   return anchorElements.map(node => {
     // @ts-ignore - put into scope via stringification
     const outerHTML = getOuterHTMLSnippet(node); // eslint-disable-line no-undef
+    // @ts-ignore - put into scope via stringification
+    const nodePath = getNodePath(node); // eslint-disable-line no-undef
+    // @ts-ignore - getNodeSelector put into scope via stringification
+    const selector = getNodeSelector(node); // eslint-disable-line no-undef
+    // @ts-ignore - getNodeLabel put into scope via stringification
+    const nodeLabel = getNodeLabel(node); // eslint-disable-line no-undef
 
     if (node instanceof HTMLAnchorElement) {
       return {
         href: node.href,
+        // TODO(blasingame): Deprecate text in favor of nodeLabel
```

Comment:
6.0 is coming up, we should do this now.

Reply:
I'm not totally convinced we want to remove this. The purpose of including some of these properties was so that plugins could take advantage of generic information that is of common interest and untruncated text seems like something useful to surface. The alt text feature of getNodeLabel is pretty nice though.

I'd say we can resolve at a later date, but @connorjclark is right that 6.0 is imminent.

Options I see, but feel free to add:

1. Add `nodeLabel` and keep `text`
1. Remove `text` and keep `nodeLabel`
1. Change `text` to just be output of `getNodeLabel`
1. Make truncateLength a parameter to `getNodeLabel` and use it as `text` without truncation

I'm a fan of 1 and 4, personally. Thoughts anyone?

Reply:
yah option 1 works for me. thanks for thinking it through

Reply:
I've removed the TODO as option 1 is already taken care of as part of this PR.